### ISSUE TITLE
ridge with normalization

### ISSUE THREAD

**Issue by SYAN83** [SYAN83/pytorch-learn#1]
ridge with normalization is different from sklearn.
Check coefficients
